src/main/java/edu/harvard/iq/dataverse/api/Files.java

```diff
@@ -1,15 +1,20 @@
 package edu.harvard.iq.dataverse.api;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonElement;
```

Comment:
Another nit: you added a bunch of imports many of which you didn't end up using

Reply:
Ah yea I'll clean those up